src/Symfony/Component/String/Uuid.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\String;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+final class Uuid implements \JsonSerializable
+{
+    public const TYPE_1 = UUID_TYPE_TIME;
+    public const TYPE_3 = UUID_TYPE_MD5;
+    public const TYPE_4 = UUID_TYPE_RANDOM;
+    public const TYPE_5 = UUID_TYPE_SHA1;
+
+    public const VARIANT_NCS = UUID_VARIANT_NCS;
+    public const VARIANT_DCE = UUID_VARIANT_DCE;
+    public const VARIANT_MICROSOFT = UUID_VARIANT_MICROSOFT;
+    public const VARIANT_OTHER = UUID_VARIANT_OTHER;
```

Comment:
What do you think about keeping constants sorted by name here?

Reply:
I used the same order as the PECL.
There are actually sorted by value:
```
    Constant [ int UUID_VARIANT_NCS ] { 0 }
    Constant [ int UUID_VARIANT_DCE ] { 1 }
    Constant [ int UUID_VARIANT_MICROSOFT ] { 2 }
    Constant [ int UUID_VARIANT_OTHER ] { 3 }
```

I prefer the current way

Reply:
Yes, that’s why I suggested sorting them by name instead.